native-code/development/prerequisite-sw/index.md

```diff
@@ -43,9 +43,11 @@ if you experience any other problems building.
 Follow step 1 and 2 at [Chromium's build instructions for Windows][4].
 
 
-### OS X
+### macOS
 
-Xcode 5 or higher. Latest Xcode is recommended to be able to build all code.
+A local installation of Xcode 9 or higher is required for builds targetting iOS.
+Latest Xcode is recommended to be able to build all code.  
+For builds targeting macOS, Xcode is bundled with the checkout.
```

Comment:
This is only true for Googlers and the bots. We're not distributing the hermetic Xcode outside of the company due to licensing reasons.

Reply:
Removed. Thanks.